app/build.gradle

```diff
@@ -3,7 +3,7 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 
-def keystorePropertiesFile = rootProject.file("keystore.properties")
+def keystorePropertiesFile = rootProject.file("keystore.properties_sample")
```

Comment:
you definitely shouldnt break app signing with this one. Keep that change local.

Reply:
You are right i forgot to change it before pull request

Reply:
@tibbi Done i deleted this change